<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageControllerClass</class>
 <widget class="QWidget" name="ImageControllerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>319</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ImageController</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="1" column="0">
    <widget class="QPushButton" name="loadButton">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="strokeButton">
     <property name="text">
      <string>Stroke</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="frameButton">
     <property name="text">
      <string>Frame</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="grayscaleButton">
     <property name="text">
      <string>Grayscale</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="quitButton">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="horCheckbox">
        <property name="text">
         <string>horizontal</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="verCheckbox">
        <property name="text">
         <string>vertical</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="normalizeCheckbox">
        <property name="text">
         <string>normalized</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="edgeSensivity">
        <property name="maximum">
         <number>150</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>70</number>
        </property>
        <property name="sliderPosition">
         <number>70</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="edgeFilter">
        <item>
         <property name="text">
          <string>Sobel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prewitt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Laplace</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="edgeButton">
        <property name="text">
         <string>Detect Edges</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="viewersCombo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>ImageControllerClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
